tsconfig.json

```diff
@@ -1,24 +1,47 @@
 {
-    "compilerOptions": {
-        "lib": ["es2017"],
-        "module": "commonjs",
-        "target": "es2017", // node 8
-        "strict": true,
-        "strictPropertyInitialization": false, // should be true
-        "esModuleInterop": true,
-        "skipLibCheck": true,
-        "forceConsistentCasingInFileNames": true,
-        "moduleResolution": "node",
-        "allowUnusedLabels": false,
-        "allowUnreachableCode": false,
-        "noFallthroughCasesInSwitch": true,
-        "noImplicitReturns": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": false, // should be true
-        "importsNotUsedAsValues": "remove", // should be error
-        "types": ["inversify", "mocha", "reflect-metadata", "restify"],
-        "experimentalDecorators": true,
-        "emitDecoratorMetadata": true,
-        "removeComments": true
-    }
-}
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "alwaysStrict": true,
```

Comment:
A lot of these added settings don't need to be explicitly added since they fall under other settings
a good example is `strict` which defaults a dozen or so rules to `true` by default.
i also think a target of `es2017` should be fine since even `restify 8.x` does not support versions of node below node 8 at least officially

Reply:
@acommodari  I'm not really sure which one is not needed, ( I added them a long time ago back in `express-utils` ) but if you have the time to sort this out, can you also please add them in `express-utils` as well ?